packages/issuer-callback/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@sphereon/openid4vci-issuer-callback",
```

Comment:
Please rename package and folder to ***callback-example

Reply:
Fixed!